locksmith/src/websub/helpers/renewFiatKey.ts

```diff
@@ -2,8 +2,7 @@ import { getStripeConnectForLock } from '../../operations/stripeOperations'
 import Dispatcher from '../../fulfillment/dispatcher'
 import { logger } from '../../logger'
 import { Charge, KeyRenewal, KeySubscription } from '../../models'
-import Stripe from 'stripe'
-import config from '../../config/config'
+import stripe from '../../config/stripe'
```

Comment:
where is the actual file?

Reply:
Another case of missing file in the git commit :D fixed!